### PR TITLE
from-dicom.ipynb has been updated

### PR DESCRIPTION
**Pull request by guanfada**

### CODE REVIEW

**Comment by SimonBiggs** [comment]
Beautiful :) here's your new PR! :)

File: lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb [hunk 1]
```diff
@@ -16,7 +16,8 @@
    "outputs": [],
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: <pre class="ql-syntax" spellcheck="false">Line #2.    class OOMFormatter(matplotlib.ticker.ScalarFormatter):
</pre><p>Is there a way to do this more simply? I'm pretty keen for these tutorials to be as easy to understand as achievable. At the end of the day, if fixing the ticker is going to be this complicated I'd prefer to leave the ticker in an untidy state for a tutorial such as this...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by guanfada** [comment]
I found I forgot to update the figure title for the gamma maps. I can do that later.

File: lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb (hunk 1 above)
Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: To see the review comment inline within the notebook click on the "ReviewNB" link:

image

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: I agree with you. I also thought this is too complicated. I found this class online.
Anyway, I will find a simpler way.

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: Yeah, ideally the tutorial would contain the minimal amount of code that achieves the desired outcome. Recognising that every piece of code in the tutorial someone new is going to be trying to understand what it is doing.

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: Will update you tomorrow. :-)
Thank you so much!

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:16]: My pleasure :) Welcome to the land of git, GitHub, and PyMedPhys :) :tada:

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb]: My pleasure :-)
I just updated again with a simpler way to show scientific notation, and updated the figure title for gamma index maps.

File: lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb [hunk 2]
```diff
@@ -168,12 +168,25 @@
     "    gamma_options['interp_fraction'] * gamma_options['max_gamma'])\n",
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:168]: <p>Can we remove this "flag probability density" instead just pick one or the other of the options and run with that?</p><p> </p><p>You can pick which one to use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:168]: Good idea. Will remove it now.

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb]: <p>"flag probability density" has been removed. You can check :-)</p>

**Review by SimonBiggs — APPROVED** [review]
Yup :), this is great :)

I'm going to approve and merge. It'll be live in the docs in ~10 mins

**Comment by guanfada** [comment]
That's great! 
I have seen the page updated.

**Pull request merged**